libraries/botbuilder-dialogs/tests/memory_memoryScopes.test.js

```diff
@@ -1,15 +1,5 @@
+const assert = require('assert');
 const { ConversationState, UserState, MemoryStorage, TurnContext, TestAdapter } = require('botbuilder-core');
-const {
-    AdaptiveDialog,
-    OnBeginDialog,
-    SendActivity,
-    LanguageGeneratorExtensions,
-    TemplateEngineLanguageGenerator,
-    RegexRecognizer,
-    IntentPattern,
-    TextInput,
-    OnIntent,
-} = require('../../botbuilder-dialogs-adaptive/lib');
```

Comment:
This is to unroll a cyclical dependency. The tests targeting all of these imports was moved into the `botbuilder-dialogs-adaptive` test suite.